### DISCUSSION TITLE
Vue 3 branch

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#6418] [Ideas] [answered]
There's a PR to port the settings page to Vue 3, but the branch is on a fork. I suggest making a Vue 3 branch on the main repo so settings page PRs aren't blocked by the port. It would also allow us to merge it and not worry about releasing it to the users in the next version. I also want to do some work on the settings page unrelated to the port but don't want to base it on the Vue 1 version.

Related:
- #6272
- #5490

**Comment by Samq64** [comment]
bump

**Reply by WorldLanguages** [reply, marked as answer]
i'm not sure if a branch is really necessary
